src/sentry/static/sentry/app/views/userFeedback/utils.jsx

```diff
@@ -41,9 +13,19 @@ export function getQuery(search) {
   const query = qs.parse(search);
 
   const status = typeof query.status !== 'undefined' ? query.status : DEFAULT_STATUS;
-  const cursor = query.cursor;
 
-  const queryParams = {status, cursor};
+  const queryParams = {
+    status,
+    ...pick(query, [
+      'cursor',
+      'project',
```

Comment:
Let's use the defined constants from `app/components/organizations/globalSelectionHeader/constants`